config/options.go

```diff
@@ -491,8 +490,8 @@ func (o *Options) Validate() error {
 		o.DataBrokerURLString = o.CacheURLString
 	}
 	switch o.DataBrokerStorageType {
-	case inmemory.Name:
-	case redis.Name:
+	case storage.InMemoryName:
```

Comment:
It feels strange to have the interface defining package to have the name of the implementing storage backends vs before (and is inconsistent with how we do things in other packages)? Can you elaborate on the change?

Reply:
Import loop from bringing in telemetry. :(

Open to other ideas, structurally.  The main issue is config is imported by a lot of other packages, so we shouldn't have it depend on much internals.

Reply:
I think we can avoid the import loop by removing runtime interface assertion, that way the redis/memory packages no longer explicitly need to import storage. 

https://github.com/pomerium/pomerium/blob/8ed40b225289f91c739132f21acf3a6dfa767167/pkg/storage/redis/redis.go#L28

Reply:
We still need to keep the assertion somewhere, maybe inside `pkg/storage/storage.go`?

Reply:
Or, put it in a test under a different package name?

Reply:
Looking again, I'm thinking the config package could just hold the names.  They're primarily a construct for the configuration file.  They're used for mapping config->backend in config and then the broker init uses the same logic.    Thoughts?